DOCUMENTATION.md

```diff
@@ -0,0 +1,196 @@
+All these folders are removed and recreated each time when geocint pipeline starts.
+
+After running the pipeline, make will create additional folders. These folders are used to store input (in folder), intermediate (mid folder), and output (out folder) data files.
+Make-profiler uses the timestamps of these files as timestamps for the execution of such a target:
+- [data/](data) - file-based input, middle, output data.
+	- data/in - all input data, downloaded elsewhere
+	- data/in/raster - all downloaded GeoTIFFs
+	- data/mid - all intermediate data (retiles, unpacks, reprojections etc.) which can be removed after
+  	each launch
+	- data/out - all generated final data (tiles, dumps, unloading for the clients etc.)
+- [db/](db) - files - Makefile mark about executing "db/..." targets
+- [deploy/](deploy) - files - Makefile mark about executing "deploy/..." targets
+- [logs/](logs) - files - files with targets execution logs
+- [report/](report) - folder to store HTML reports.
+
+These folders are not deleted or re-created each time the geocint pipeline is run, to avoid rebuilding targets that should not be rebuilt every time (if you want to rebuild some targets chain each time please see How geocint pipeline).
+You shouldn’t store all your input datasets in data/in/ folder. To make your data storage more organized, you can create additional folders for separate data sources (for example data/in/source_name). This also applies to other catalogs.
+
+## Geocint open-source installation and first run guide
+
+### Intallation
+Before to install your own geocint pipeline instance, you should create a repository to store your own part of the pipeline.
+Your repository should contain the following required files:
+- README.md (could be empty, just make sure that it exists)
+- install.sh (use [runner_install.sh](runner_install.sh) as an example, store installation of your additional dependencies)
+- your_make (use [private_make.sample](your_make.sample) as an example; file to store your own additional targets chains. Keep in mind that make shouldn't be named "Makefile", use the other name to keep compatibility with geocint-runner repository)
+
+1. Create a new user with sudo permissions or use existing (the default user is "gis"). Keep in mind that the best practice is to use this username for creating a Postgres role and database. Path ~/ is equivalent to /home/your_user/. This folder is a working directory for the geocint pipeline.
+2. Clone 3 repository (geocint-runner, geocint-openstreetmap, your repo) to ~/
+3. The geocint pipeline should send messages to the Slack channel. Create a channel, generate Slack token and store it in the `SLACK_KEY` variable in file `~/.profile`.
+```shell
+export SLACK_KEY=<your_key>
+```
+4. Copy [config.inc.sh.sample](config.inc.sh.sample) from geocint-runner to ~/:
+```shell
+cp ~/geocint-runner/config.inc.sh.sample ~/config.inc.sh
+```
+Open ~/config.inc.sh and set the necessary values of variables.
```

Comment:
maybe move about SLACK_KEY in this section?

Reply:
added comment "# this script is using one during installation to create /public_html and /domlogs folders and set access modes"

Reply:
@lgudyma-kontur what do you mean

Reply:
there is **3.** about setting slack_key, and later you there is setting vars in config.inc.sh... maybe join them in one